src/Uno.SourceGeneration.Host/Program.cs

```diff
@@ -1,96 +1,37 @@
-﻿using System;
+﻿using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Logging.Console;
+using System;
 using System.Diagnostics;
-using System.IO;
 using System.Linq;
-using System.Reflection;
-using System.Runtime.Serialization;
-using Microsoft.Build.Framework;
-using Microsoft.Build.Logging;
-using Microsoft.Extensions.Logging;
-using Microsoft.Extensions.Logging.Console;
-using Uno.SourceGeneration.Helpers;
-using Uno.SourceGeneratorTasks;
+using Uno.SourceGeneration.Host.Helpers;
+using Uno.SourceGeneration.Host.Server;
 using Uno.SourceGeneratorTasks.Helpers;
 
+[assembly: CommitHashAttribute("<developer build>")]
```

Comment:
What is that??

Reply:
It's to make sure that the client is connected to a compatible server. Which reminds me I need to change it in the builds scripts.